frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.tsx

```diff
@@ -10,68 +16,153 @@ import { color } from "metabase/lib/colors";
     <Popover
       position="bottom-start"
-      opened={suggestions?.length > 0}
+      opened={suggestions.length > 0}
       radius="xs"
       withinPortal
       zIndex={300}
-      returnFocus
     >
       <Popover.Target>{children}</Popover.Target>
       <Popover.Dropdown>
         <DelayGroup>
-          <ExpressionList
-            data-testid="expression-suggestions-list"
-            className={CS.pb1}
-          >
-            {suggestions.map((suggestion: Suggestion, idx: number) => (
-              <ExpressionEditorSuggestionsListItem
-                key={`suggesion-${idx}`}
-                query={query}
-                stageIndex={stageIndex}
-                suggestion={suggestion}
-                isHighlighted={idx === highlightedIndex}
-                index={idx}
-                onMouseDown={onSuggestionMouseDown}
-              />
-            ))}
+          <ExpressionList data-testid="expression-suggestions-list">
+            <ExpressionEditorSuggestionsListGroup
+              suggestions={groups._none}
+              query={query}
+              stageIndex={stageIndex}
+              highlightedIndex={highlightedIndex}
+              onSuggestionMouseDown={onSuggestionMouseDown}
+            />
+            <ExpressionEditorSuggestionsListGroup
+              name="popularAggregations"
+              suggestions={groups.popularAggregations}
+              query={query}
+              stageIndex={stageIndex}
+              highlightedIndex={highlightedIndex}
+              onSuggestionMouseDown={onSuggestionMouseDown}
+            />
+            <ExpressionEditorSuggestionsListGroup
+              name="popularExpressions"
+              suggestions={groups.popularExpressions}
+              query={query}
+              stageIndex={stageIndex}
+              highlightedIndex={highlightedIndex}
+              onSuggestionMouseDown={onSuggestionMouseDown}
+            />
           </ExpressionList>
         </DelayGroup>
+        {footers.map(suggestion => (
+          <Footer
+            key={suggestion.index}
+            suggestion={suggestion}
+            highlightedIndex={highlightedIndex}
+          />
+        ))}
       </Popover.Dropdown>
     </Popover>
   );
 }
 
+function ExpressionEditorSuggestionsListGroup({
+  name,
+  query,
+  stageIndex,
+  suggestions = [],
+  onSuggestionMouseDown,
+  highlightedIndex,
+}: {
+  name?: GroupName;
+  query: Lib.Query;
+  stageIndex: number;
+  suggestions?: Suggestion[];
+  onSuggestionMouseDown: (index: number) => void;
+  highlightedIndex: number;
+}) {
+  const definition = name && GROUPS[name];
+
+  if (suggestions.length === 0) {
+    return null;
+  }
+
+  return (
+    <>
+      {definition?.displayName && (
+        <GroupTitle isHighlighted={false}>{definition.displayName}</GroupTitle>
+      )}
+      {suggestions.map((suggestion: SuggestionWithIndex) => (
+        <ExpressionEditorSuggestionsListItem
+          key={`suggesion-${suggestion.index}`}
```

Comment:
Not that it matters but:

```suggestion
          key={`suggestion-${suggestion.index}`}
```